From b6d5b8fec7731fd3bcfcc579f19dbd8a85a169cf Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Fri, 17 Feb 2017 10:11:35 +0800
Subject: [PATCH 1528/5242] MLK-13991: ASoC: fsl_rpmsg_i2s: remove unsupported
 rate

commit  bec41c93f49a2116cd86129e670169b1d5df7a42 from
https://source.codeaurora.org/external/imx/linux-imx.git

The M4 audio driver only support 8k/16k/32k/44k/48kHz sample rate.
so remove other rate in supported list.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_rpmsg_i2s.c |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/sound/soc/fsl/fsl_rpmsg_i2s.c b/sound/soc/fsl/fsl_rpmsg_i2s.c
index c53807f..172c279 100644
--- a/sound/soc/fsl/fsl_rpmsg_i2s.c
+++ b/sound/soc/fsl/fsl_rpmsg_i2s.c
@@ -26,7 +26,9 @@
 #include "fsl_rpmsg_i2s.h"
 #include "imx-pcm.h"
 
-#define FSL_RPMSG_I2S_RATES	SNDRV_PCM_RATE_8000_96000
+#define FSL_RPMSG_I2S_RATES	(SNDRV_PCM_RATE_8000|SNDRV_PCM_RATE_16000|\
+				SNDRV_PCM_RATE_32000|SNDRV_PCM_RATE_44100|\
+				SNDRV_PCM_RATE_48000)
 #define FSL_RPMSG_I2S_FORMATS	SNDRV_PCM_FMTBIT_S16_LE
 
 static int i2s_send_message(struct i2s_rpmsg_s *msg,
-- 
1.7.9.5

